main.go

```diff
@@ -30,12 +30,14 @@ var logFormat = flag.String("log-format", "json", "can be empty string or json")
 var logLevel = flag.String("log-level", "info", "Can be one of:"+strings.Join(validLogLevels(), ","))
 var templateString = flag.String("template-string", messageTemplate, "template for the messages sent to hangouts chat")
 
-var messageTemplate = `<users/all>
-*{{.QueryParams.Get "env" | toUpper }}: {{ .Labels.alertname }} - {{.Status | toUpper}}*
+var messageTemplate = `*{{ .Labels.alertname }} - {{.Status | toUpper}}*
+{{ range .Labels.SortedPairs -}}
```

Comment:
Just for my interest: can you please explain me what's the benefit of this sort?

Reply:
> Just for my interest: can you please explain me what's the benefit of this sort?

I think no additional benefit when it sorted, is there any good practice to iterate .Labels?